03-Node-npm-modules/js/lib/Director.js

```diff
@@ -0,0 +1,21 @@
+var Director = function (name) {
+    this.name = name;
+    this.quotes = [];
+};
+
+Director.prototype = {
+  contructor : Director, 
+  get : function (property) {
+    return this[property];
+  }, 
+  set :function (property, value) {
+    this[property] = value; ;
+  },
+  speak : function () {
+    var sentence = '';
+    sentence = this.quotes.join(',')
+    console.log(this.name + ' says :' + sentence);
+  }
+};
+
+exports.Director = Director;
```

Comment:
if you do `module.exports = Director` from the require you'll only need to do `require('./director);` inside the movie.js file